### PR TITLE
Creating a data model

### PR DESCRIPTION
**Pull request by eerapheal**
This Blog app is created with  Ruby on Rails with the objectives below.

Install Ruby on Rails framework.
Use Ruby Gems as a software package system.
Understand what ORM is.
Use database migration files to maintain database schema.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Ralph, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe/repository of this project adhere(s) to the following standard(s):
- [ ] All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the FAQ section is optional.
- [ ]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- [ ] Kindly ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] Kindly ensure that provided text instructions are removed and replaced with appropriate contents applicable under each section of your ReadMe.
- [ ] Kindly ensure to use the same Pull Request as the first time this activity was reviewed to request for another code review, so as to ensure review consistency.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,116 @@
...
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
```
Comment (Reem-lab) [inline README.md:23]: - [ OPTIONAL ] I highly recommend adding a description to your README it is very important to add a description to explain what your project about and what is for so kindly add it 🙏🏻🔥✅✨

**Review by Reem-lab — APPROVED** [review]
# Hi @eerapheal, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Followed Gitflow ✅
- Descriptive PR ✅
- Great working on the functionality ✅
-  Professional README ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (eerapheal) [inline README.md:23]: Thanks for the review @Reem-lab.

**Pull request merged**